math/src/field/fields/p448_goldilocks_prime_field.rs

```diff
@@ -209,12 +247,18 @@ impl U56x8 {
     }
 }
 
+impl FieldElement<P448GoldilocksPrimeField> {
+    pub fn new_base(a_hex: &str) -> Self {
+        Self::new(U56x8::from(a_hex))
+    }
+}
+
```

Comment:
This should be named `from_hex`. Since this field is prime and `from_hex` is one of the methods of the trait `IsPrimeField`, it seems like the most sensible thing to do here is implement that trait for `P448GoldilocksPrimeField`. Use `U448` as `RepresentativeType`.

Reply:
Done.